src/dolphin/gx/GXTransform.h

```diff
@@ -1,17 +1,19 @@
 #ifndef DOLPHIN_GX_GXTRANSFORM_H
 #define DOLPHIN_GX_GXTRANSFORM_H
 
-#include <dolphin/types.h>
+#include <dolphin/mtx/mtxtypes.h>
 
 void GXProject(
-	f32 x, // model coordinates
-	f32 y,
-	f32 z,
-	f32 mtx[3][4], // model-view matrix
-	f32 *pm,	   // projection matrix, as returned by GXGetProjectionv
-	f32 *vp,	   // viewport, as returned by GXGetViewportv
-	f32 *sx,	   // screen coordinates
-	f32 *sy,
-	f32 *sz);
+    f32 x, // model coordinates
+    f32 y,
+    f32 z,
+    MtxPtr mtx, // model-view matrix
```

Comment:
Not a good change, this is a functional difference.

Reply:
Can `Mtx` be used? I don't really understand how C typedefs work in this case because `Mtx` is literally `f32[3][4]`.

Reply:
Mtx could be used, yeah.